Assets/Scripts/DataController.cs

```diff
@@ -57,12 +57,12 @@ public List<ConstellationConnection> AllConstellationConnections
 
     public bool showHorizonView = false;
 
-    public GameObject starPrefab;
+    public StarComponent starPrefab;
 
     public GameObject allConstellations;
     private ConstellationsController constellationsController;
     public List<string> constellationFullNames;
-    public GameObject constellationPrefab;
+    public Constellation constellationPrefab;
```

Comment:
I've always liked the more traditional pattern of typing a prefab for instantiation as a `GameObject`.  I'm guessing this is to save on `GetComponent` calls to pull the MonoBehavior off of the instantiated game object?  Seems to work fine.

Reply:
It's a few lines saved here and there, and since they're all MonoBehaviours, they all have a .gameObject when we need it. Can't say how much this saves us in performance, but it simplifies things and reduces the number of temporary object references, so it can't hurt!